lib/alephant/broker/response/base.rb

```diff
@@ -48,10 +52,19 @@ def add_no_cache_headers
           log
         end
 
+        def add_etag_allow_header
+          headers.merge!("Access-Control-Expose-Headers" => "ETag")
+        end
+
         def component_not_modified(headers, request_env)
```

Comment:
@dazoakley, I have added static methods now for the unmodified response check, and also the etag method. They are quite long now, as I have to prefix the class name.